src/Core/Content/Category/Service/CategoryBreadcrumbBuilder.php

```diff
@@ -20,12 +22,20 @@ class CategoryBreadcrumbBuilder
      */
     private $categoryRepository;
 
+    /**
+     * @var EventDispatcherInterface
+     */
+    private $dispatcher;
+
     /**
      * @deprecated tag:v6.4.0.0 - EntityRepositoryInterface will be required
      */
-    public function __construct(?EntityRepositoryInterface $categoryRepository = null)
-    {
+    public function __construct(
+        ?EntityRepositoryInterface $categoryRepository = null,
+        EventDispatcherInterface $dispatcher
```

Comment:
This is not allowed in PHP 8. 
https://php.watch/versions/8.0/deprecate-required-param-after-optional